### PR TITLE
Disperse the event files into different sub-folders according to the prefix of file name.

### PR DESCRIPTION
**Pull request by edwardmeng**
When the files count more than 100,000 then performance of hard disk will becomes worse, and there are no more than 10,000 files can get the best result. So disperse the files into different sub-folders can improve the performance and stability.

### CODE REVIEW

**Comment by niemyjski** [comment]
Looks good, bit worried about a ton of folders being created in the low throughput case (36*36*36, wonder if two characters would still provide the benefit). Could you also update GetArchivePath to include the minute in the archive path.

**Comment by edwardmeng** [comment]
Please take review again

File: src/Exceptionless.Web/Properties/launchSettings.json [hunk 1]
```diff
@@ -12,7 +12,7 @@
       "commandName": "IISExpress",
       "launchBrowser": true,
       "environmentVariables": {
-        "AppMode": "Development"
+        "AppMode": "Production"
```
Comment (niemyjski) [inline src/Exceptionless.Web/Properties/launchSettings.json]: Can you revert this change please :)

**Comment by edwardmeng** [comment]
Please take review again.

**Comment by niemyjski** [comment]
Thanks for the pr
